src/NuGet.Clients/NuGet.VisualStudio.Common/ScriptExecutionRequest.cs

```diff
@@ -43,11 +43,12 @@ public ScriptExecutionRequest(string scriptPath, string installPath, PackageIden
 
         public string BuildCommand()
         {
-            var escapedScriptPath = PathUtility.EscapePSPath(ScriptPath);
             var command = new StringBuilder(
                 "$__pc_args=@(); " +
                 "$input|%{$__pc_args+=$_}; " +
-                "& " + escapedScriptPath + " $__pc_args[0] $__pc_args[1] $__pc_args[2]");
```

Comment:
Should these raw string concatenations be avoided?  In other words, why not the following?
```C#
var command = new StringBuilder("$__pc_args=@(); ");
command.Append("$input|%{$__pc_args+=$_}; ");
command.Append("& ");
…
```
Or even:
```C#
var command = new StringBuilder("$__pc_args=@(); $input|%{$__pc_args+=$_}; & ");
…
```

Reply:
The C# compiler will do this optimization automatically. You can verify this using a decompiler 😄